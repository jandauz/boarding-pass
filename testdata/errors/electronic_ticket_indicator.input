M1DESMARAIS/LUC       AABC123 YULFRAAC 0834 326J001A0025 100