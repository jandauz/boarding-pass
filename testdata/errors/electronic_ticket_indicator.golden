bcbp: Invalid data format:
  boarding pass data:
  | "M1DESMARAIS/LUC       AABC123 YULFRAAC 0834 326J001A0025 100"
  |                        ^ got "A"
  |
  = reason: data for "Electronic Ticket Indicator" must be E or L
